spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployer.java

```diff
@@ -197,6 +202,23 @@ public void shutdown() throws Exception {
 		}
 	}
 
+	private String buildRemoteDebugInstruction(Map<String, String> deploymentProperties) {
+		String dp = deploymentProperties.get(LocalDeployerProperties.DEBUG_PORT);
+		String ds = deploymentProperties.getOrDefault(LocalDeployerProperties.DEBUG_SUSPEND, "y");
+		StringBuilder debugCommandBuilder = new StringBuilder();
+		if (StringUtils.hasText(dp)) {
+			logger.warn("***** [REMOTE DEBUGGING] REMOTE DEBUGGING IS ENABLED ON PORT " + dp + " *****");
```

Comment:
The `deploymentId` should be passed into this method to give context to these warning messages, otherwise one can't tell what app is having remote debugging enabled.

Reply:
Actually, while doing some refactoring, this property should also be available for when deploying Tasks....so it should move up into the `AbstractLocalDeployerSupport` class and `LocalTaskLauncher` can take advantage of it as well.

Reply:
I'll push a variation on this to a new branch.